Release Notes/511.md

```diff
@@ -16,6 +16,12 @@
 - Effect specifiers:
   - Description: The `unexpectedAfterThrowsSpecifier` node of the various effect specifiers has been removed.
   - Pull request: https://github.com/apple/swift-syntax/pull/2219
+- `SyntaxKind` removed conformance to `CaseIterable`
+  - Description: `SyntaxKind` no longer conforms to `CaseIterable` since there is no good use case to iterate over all syntax kinds. 
+  - Pull request: 
```

Comment:
I'm not sure if you've forgotten or are planning to do this next, but the Release Notes are missing the PR link. Here's the suggestion:
```suggestion
  - Pull request: https://github.com/apple/swift-syntax/pull/2292
```